src/Compare.js

```diff
@@ -26,6 +27,9 @@ class Compare extends Component {
     e.target.classList.remove('up')
```

Comment:
I believe the only addition to this was the carats next to the name for filtering. I see that you added a conditional for add, sort and remove